custom/icds_reports/reports/adolescent_girls.py

```diff
@@ -64,24 +67,27 @@ def get_data_for(filters):
         "fills": fills,
         "rightLegend": {
             "average": '%.2f' % (total_valid * 100 / float(total or 1)),
-            "info": percent_adolescent_girls_enrolled_help_text(),
+            "info": percent_adolescent_girls_enrolled_help_text_v2() if beta else
```

Comment:
This changes map

Reply:
![ps_map](https://user-images.githubusercontent.com/21310920/71279545-0f231e80-237f-11ea-9314-a8c331c0a46e.png)